app/assets/json/translations/locale-es.json

```diff
@@ -454,7 +454,7 @@
         "not_found_body": "Estimado(a) MOHCD,\n\nno puedo completar mi solicitud para {{listing_name}}. Mi dirección es {{home_address}} y el formulario me señala que hay un error. Indíqueme qué puedo hacer a continuación.\n\nGracias,\n{{first_name}} {{last_name}}\n{{email}}\n{{phone_number}}",
         "not_found_subject": "No puedo ingresar mi dirección en la solicitud en DAHLIA"
       },
-      "address_validation_duplicate_unit": "Si tiene un número de unidad o departamento, simplemente ingréselo en el casillero “Apt or Unit #” (N.° de departamento o unidad)",
+      "address_validation_duplicate_unit": "Si tiene un número de unidad o departamento, simplemente ingréselo en el casillero Apartamento o unidad n.º",
```

Comment:
is that special character supposed to be there at the end?

Reply:
The recommendation was to not use abbreviation. Just pushed a fix.